Source/WarpX.cpp

```diff
@@ -730,9 +730,9 @@ WarpX::ReadParameters ()
         }
         else // FDTD
         {
-            // Filter currently not working with FDTD solver in RZ geometry
+            // Filter currently not working with FDTD solver in RZ geometry along R
             // (see https://github.com/ECP-WarpX/WarpX/issues/1943)
-            WARPX_ALWAYS_ASSERT_WITH_MESSAGE(!use_filter,
+            WARPX_ALWAYS_ASSERT_WITH_MESSAGE(!use_filter || filter_npass_each_dir[0] == 0,
                 "Filter currently not working with FDTD solver in RZ geometry");
```

Comment:
Could you modify the error message to say that filtering only works along z for the FDTD solver in RZ?
Could the error message also tell the user how to set filtering along z only in their input script (using `n_pass_each_dir`)?

Reply:
I added filtering in z in the LaserAccelerationRZ test and updated the benchmark.